.gitignore

```diff
@@ -0,0 +1,2 @@
+Examples/config.ini
+*.swp
```

Comment:
This PR contains too many things done at once in one commit.

My suggestions:
  - implement sdelete in one PR
  - gitignore, examples and other fixes should go in a separate PR

Reply:
Ok